simEarth_fork.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+#state = np.array([100, 200, 130, 1.05, 1.02, 0.75])
+
+    state = {"pop": 100 , "fos"; 200, "fol": 130, "bir": 1.05, "der": 1.02, "fef": 0.75}
+
+def function_food_growth(array):
+    food_supply = array[fos]
+    fertility_of_land = array[fol]
+    population = array[pop]
+    fertility_factor = array[fef]
+    new_food = (fertility_of_land - fertility_factor * population) + food_supply
+    array[fos] = [new_food]
```

Comment:
In this line you assign `array[fos]` to an array/list with the entry new_food. Is this intended?